### PR TITLE
Kafka exporters

### PR DESCRIPTION
**Pull request by bvdeenen**
Has commands for Kafka Exporters and Kafka Users

### CODE REVIEW

File: src/main/kotlin/io/streammachine/api/cli/commands/KafkaExporters.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            KafkaExportersCreate(),
+            KafkaExportersDelete()
+        )
+    }
+
+    override fun run() = Unit
+}
+
+open class KafkaExportersGet : CliktCommand(
+    name = "get",
```
Comment (trietsch) [inline src/main/kotlin/io/streammachine/api/cli/commands/KafkaExporters.kt:37]: We use the verb `list` elsewhere in the CLI. Though that implies getting a list, not a single exporter. Knowing that we'll create a version 2 soon for the CLI, do you agree to leave it like this?

File: src/main/kotlin/io/streammachine/api/cli/commands/KafkaExportersUsers.kt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            KafkaExportersUsersDelete()
+        )
+    }
+
+    override fun run() = Unit
+}
+
+open class kafkaExportersUsersList : CliktCommand(
+    name = "list",
+    help = "get all users on a certain Kafka exporter"
```
Comment (trietsch) [inline src/main/kotlin/io/streammachine/api/cli/commands/KafkaExportersUsers.kt:29]: Capitalize `get`.

File: src/main/kotlin/io/streammachine/api/cli/commands/KafkaExportersUsers.kt [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    override fun run() {
+        "/v1/kafka-exporters-users/$streamName".httpGet().printResponse()
+    }
+}
+open class KafkaExportersUsersGetOne : CliktCommand(
+    name = "get",
+    help = "Get a specific user on a Kafka exporter by stream name"
+) {
+    internal val streamName by argument("stream-name", help = "Name of the stream that is being exported")
+    internal val clientId by argument("client-id", help = "The client/consumer id that identifies the user")
```
Comment (trietsch) [inline src/main/kotlin/io/streammachine/api/cli/commands/KafkaExportersUsers.kt:42]: How come this is a `consumer id`? It's the client id for this stream, that the customer uses to authenticate their app with Stream Machine in order to start consuming data, right? Hence, it's a client credentials flow in OAuth terms.

**Review by trietsch — COMMENTED** [review]
Looks good overall! Small remarks made.

File: src/main/kotlin/io/streammachine/api/cli/commands/KafkaExporters.kt (hunk 1 above)
Comment (bvdeenen) [reply src/main/kotlin/io/streammachine/api/cli/commands/KafkaExporters.kt:37]: get indeed gets 0 or 1, so get seems more appropriate

**Pull request merged**